src/gromacs/nblib/gmxsetup.h

```diff
@@ -71,13 +71,27 @@ struct NbvSetupUtil
 {
     NbvSetupUtil(SimulationState system, const NBKernelOptions& options);
 
+    //! Sets up and returns a GmxForceCalculator
+    std::unique_ptr<GmxForceCalculator> setupGmxForceCalculator();
+
+private:
```

Comment:
If this has private members it should be a class.

Reply:
Done.